src/azure-cli/azure/cli/command_modules/billing/generated/action.py

```diff
@@ -0,0 +1,106 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=protected-access
+
+import argparse
+from collections import defaultdict
+from knack.util import CLIError
+
+
+class AddSoldTo(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+
+        if hasattr(namespace, 'sold_to'):
+            namespace.sold_to = action
+        if hasattr(namespace, 'bill_to'):
+            namespace.bill_to = action
```

Comment:
Don't change code in "generated" folder and we should override action in "manual" folder so that it will not be overwritten next time.

Reply:
done. Moved to manual/